### DISCUSSION TITLE
Icon request: fa-attic

### DISCUSSION THREAD

**Discussion by paigemillett** [FortAwesome/Font-Awesome#21192] [Icon Requests]
Working on a real estate app and there are no good icons for an attic. Something along the lines of just a roof with a window, or maybe the trap door that is often used for attics

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
